clap_derive/src/attrs.rs

```diff
@@ -782,6 +784,18 @@ impl Attrs {
         quote!( #(#doc_comment)* #(#methods)* )
     }
 
+    /// generate methods on top of a field
+    /// filter out any passed method names
+    pub fn field_methods_filtered(&self, filter: &[&str]) -> proc_macro2::TokenStream {
```

Comment:
I'd clarify this is intended for derives where an attribute that normally maps to a derive doesn't (with `alias` being the example).